CMakeLists.txt

```diff
@@ -12,9 +12,14 @@ if (PROXIES_ENABLE_WARNINGS)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Werror")
 endif()
 
+if (ENABLE_CUDA)
+  set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --expt-extended-lambda")
```

Comment:
I understand why this has added, but how did it work before?! I think that including the RAJA submodule used to update the flags.